concepticondata/references/references.bib

```diff
@@ -2099,3 +2099,16 @@ @Phdthesis{Yang2011
   Note                                = {PhD Thesis},
   Address                             = {Bundoora}
 }
+
+@Article{Vejdemo2016,
+  Title                               = {{S}emantic {F}actors {P}redict the {R}ate of {L}exical {R}eplacement of {C}ontent {W}ords},
+  Author                              = {Vejdemo, Susanne and Hörberg. Thomas},
+  Journal                             = {PLoS ONE},
+  Year                                = {2016},
+  Volume                              = {11},
+  Number                              = {1},
+  Doi                                 = {10.1371/journal.pone.0147924},
+  Url                                 = {https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0147924#pone.0147924.ref032}
+}
```

Comment:
we add page numbers in journals like plos as well, just use the page numbers of the manuscript.

Reply:
sorry, but this is not what I meant: I meant that you also need to write them as small, obviously: bibtex uses the `{T}` to indicate capitalization, but our systems ignore this, so you need to write in small capitals to get this right, as it will still be a capitalized title, which is not the standard style. It is possible that our bibliography is inconsistent. I go through it once a while, but this can obviously be avoided, if we do it more carefully generally upon adding a new list. But if this is done, you can merge it

Reply:
Obviously, the pages are also still missing, right? Just use the page nmbers in hte paper: 1-15. Again, one could say this is nitpicking, but we are linguists, and we need to insist on good bibliographies, physics or biologists can do without it, if they prefer, but clean bibliographies are part of our standard. So just add the page-range (1-15, obviously), and this is fine.